.rubocop.yml

```diff
@@ -85,15 +85,9 @@ RSpec/NoExpectationExample:
 Capybara/SpecificFinders:
   Enabled: false
 
-Capybara/CurrentPathExpectation:
-  Enabled: false
```

Comment:
This doesn't change anything, right?

Reply:
Ah OK I've missed them! Thanks for the clarification

I just see that you actually made single commits for every cop 👏🏽 👏🏽